Strategies/EMA.py

```diff
@@ -0,0 +1,42 @@
+from exchanges.gdax import CandleList
+class EMA():
```

Comment:
do you happen to have a general description or link that would explain what this all is.